src/cc/export/helpers.h

```diff
@@ -189,8 +189,8 @@ __attribute__((section("maps/export"))) \
 struct _name##_table_t __##_name
 
 // define a table that is shared across the programs in the same namespace
-#define BPF_TABLE_SHARED(_table_type, _key_type, _leaf_type, _name, _max_entries) \
-BPF_TABLE(_table_type, _key_type, _leaf_type, _name, _max_entries); \
+#define BPF_TABLE_SHARED(_table_type, _key_type, _leaf_type, _name, _max_entries, _flags) \
+BPF_F_TABLE(_table_type, _key_type, _leaf_type, _name, _max_entries, _flags); \
```

Comment:
This won't work. I would like both BPF_TABLE_SHARED(_table_type, _key_type, _leaf_type, _name, _max_entries)  and BPF_TABLE_SHARED(_table_type, _key_type, _leaf_type, _name, _max_entries, _flags) works.
See BPF_TABLE_PINNED example.
```
$ egrep -r BPF_TABLE_PINNED
cc/test_pinned_table.cc:      BPF_TABLE_PINNED("hash", u64, u64, ids, 0, "/sys/fs/bpf/test_pinned_table", BPF_F_NO_PREALLOC);
cc/test_pinned_table.cc:      BPF_TABLE_PINNED("hash", u64, u64, ids, 1024, "/sys/fs/bpf/test_pinned_table");
cc/test_pinned_table.cc:      BPF_TABLE_PINNED("sk_storage", __u32, __u64, sk_stg, 0, "/sys/fs/bpf/test_pinned_table");
```